addons/stock/report/report_deliveryslip.xml

```diff
@@ -78,16 +78,10 @@
                                 <td class="text-end">
                                     <span t-field="move.product_uom_qty">3.00</span>
                                     <span t-field="move.product_uom" groups="uom.group_uom">units</span>
-                                    <span t-if="move.product_packaging_id">
```

Comment:
Doesn't it still have a `packaging_uom_id`? Warehouse people doing the delivery/reception won't bother on what's inside a box, they'll want to know how many boxes there should be.